README.md

```diff
@@ -1,2 +1,146 @@
-# atoma-agents
-Agentic workflows leveraging the Atoma Network
+# Atoma Agents
+
+A powerful, extensible AI agent system for blockchain operations, currently supporting the Sui blockchain. This project aims to provide a comprehensive solution for interacting with various blockchain protocols through natural language processing.
```

Comment:
Let's start by first referring to the Sui blockchain, also emphasize the Atoma Network